python/measurement/average_taxons.py

```diff
@@ -0,0 +1,20 @@
+import gzip
+from lib import json_arrays
+from lib.helpers import dig
+from lib import services
+from statistics import mean
+from data_extraction.taxonomy_query import TaxonomyQuery
+
+
+def measure_average_taxons(filename):
+
+    __query = TaxonomyQuery()
+
+    def __number_of_taxons(content_item):
+        taxons = dig(content_item, 'links', 'taxons') or []
+        return len([taxon for taxon in taxons if __query.taxon_linked_to_root(taxon)])
+
+    with gzip.open(filename, mode='rt') as file:
+        content_generator = json_arrays.read_json(file)
+        value = mean(map(__number_of_taxons, content_generator))
```

Comment:
Thinking about the usefulness of this, I think it would be good to exclude content items not tagged to the taxonomy, as I think that metric has more information in it.

That would require filtering the result of map here to remove the 0 values.

Reply:
done